dist/obs-server.spec

```diff
@@ -297,6 +297,10 @@ This package contains all the necessary tools for upload images to the cloud.
 
 %setup -q -n open-build-service-%version
 
+# We don't need our docker files in our packages
+rm -r src/{api,backend}/docker-files
```

Comment:
@bgeuken, I think is missing `-f` option, right?

Reply:
I would avoid forcing it, unless it's needed. I tried it in my [branch](https://build.opensuse.org/package/show/home:bgeuken:branches:OBS:Server:Unstable/obs-server) and it worked without.